lab-davidw/routes/bait-routes.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
```

Comment:
One of your endpoints should show a relationship between resources. Ex. /lures/:luresId/bait/:id'